devices/debian_wifi.py

```diff
@@ -17,3 +18,54 @@ def scan(self):
         self.expect(self.prompt)
 
         return self.before
+
+    def wpa_connect(self,ssid_name,pass_word,wlan_intf):
+
+         '''Generate WPA supplicant file and execute it'''
+
+         self.sendline("rm /etc/"+ssid_name+".conf")
+         self.expect(self.prompt)
+         self.sendline("wpa_passphrase "+ssid_name+ " >> /etc/"+ssid_name+".conf")
+         self.expect("")
```

Comment:
i am guessing there is no prompt of any kind being displayed....

Reply:
Once we pass the "wpa_passphrase" command it waits for password to be entered. Hence in this case we don't expect any prompt.